src/display/api.js

```diff
@@ -152,6 +152,9 @@ function setPDFNetworkStreamFactory(pdfNetworkStreamFactory) {
  *   parsed font data from the worker-thread. This may be useful for debugging
  *   purposes (and backwards compatibility), but note that it will lead to
  *   increased memory usage. The default value is `false`.
+ * @property {HTMLDocument} [ownerDocument] - Specify an explicit document
+ *   context to create elements with and to load resources into, such as
+ *   fonts. Defaults to the current document.
```

Comment:
This *might* flow a little better with:
```suggestion
 * @property {HTMLDocument} [ownerDocument] - Specify an explicit document
 *   context to create elements with and to load resources, such as fonts,
 *   into. Defaults to the current document.
```

Reply:
Furthermore, please add a default value [after this line](https://github.com/mozilla/pdf.js/blob/c5663f2f6b95680c9b7ce0c0f1e0021aa22deebe/src/display/api.js#L270), similar to the other `getDocument` options:
```javascript
if (typeof params.ownerDocument === "undefined") {
  params.ownerDocument = globalThis.document;
}
```